os/net/lwip/src/core/netif.c

```diff
@@ -1220,9 +1220,11 @@ err_t netif_gen_stable_private_id(struct netif *netif, s8_t addr_idx, ip6_addr_t
 		param.mac[i] = netif->hwaddr[i];
 	}
 	// ToDo : mbedTLS is in userspace. so mbedtls_sha256 can't be called.
+/*
```

Comment:
I think this is related with what you said above. Even it is, you should not block tizenrt common code like this. Please revert this change.